report_generator/run_siege_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+echo "Script usage: sh run_siege_tests.sh RANDOM_SEED CONCURRENT_USERS "
```

Comment:
echo and exit should be indented.
RANDOM_SEED and CONCURRENT_USERS could have <> braces around them to distinguish mandatory arguments (as opposed to [] braces that denote optional arguments).